stmhal/Makefile

```diff
@@ -321,6 +321,16 @@ $(BUILD)/%_$(BOARD).c $(HEADER_BUILD)/%.h $(HEADER_BUILD)/%_af_const.h $(BUILD)/
 $(BUILD)/pins_$(BOARD).o: $(BUILD)/pins_$(BOARD).c
 	$(call compile_c)
 
+GEN_STMCONST_HDR = $(BUILD)/modstm_const.gen.h
```

Comment:
> It wasn't a standalone header, so that's why I was leaning to having it out.

True, but it is included in modstm.c.

> We should probably move pins_qstr.h into genhdr as well to be consistent

But this (and modstm_qstr.h) are *not* included in any .c file (or .h file).  They are just intermediate files for qstr processing.